sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java

```diff
@@ -3,14 +3,25 @@
 import com.google.errorprone.annotations.Var;
 import com.google.protobuf.ByteString;
 import com.google.protobuf.InvalidProtocolBufferException;
-import com.hedera.hashgraph.sdk.proto.*;
+//import com.hedera.hashgraph.sdk.proto.*;
```

Comment:
We can remove this now.

Reply:
I need to start looking at my diffs before I commit